### DISCUSSION TITLE
have issues to link openssl

### DISCUSSION THREAD

**Discussion by defnax** [Homebrew/discussions#1089] [Everyday usage]
Hi,
im more new to MacOS and I installed openssl without issues with Brew.
My System MacOs High Sierra (10.13.6), Xcode 10.1
I have issues on compiling that the openssl lib is not found.

I used 
$ brew link --force openssl not helped 
and export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib" too to my env

```
$ brew link --force openssl
Warning: Refusing to link macOS provided/shadowed software: openssl@1.1
If you need to have openssl@1.1 first in your PATH, run:
  echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> ~/.profile

For compilers to find openssl@1.1 you may need to set:
  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"

For pkg-config to find openssl@1.1 you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
```

**ENV**

> $ printenv
> LDFLAGS=-L/usr/local/opt/openssl@1.1/lib
> TERM_PROGRAM=Apple_Terminal
> SHELL=/bin/bash
> TERM=xterm-256color
> CPPFLAGS=-I/usr/local/opt/openssl@1.1/include
> TMPDIR=/var/folders/gv/1w88v44d5r33tvs3qv2m94wh0000gn/T/
> Apple_PubSub_Socket_Render=/private/tmp/com.apple.launchd.arfR7pKr7R/Render
> TERM_PROGRAM_VERSION=404
> OLDPWD=/Users/test
> TERM_SESSION_ID=188BFFE2-0A5A-4E6C-8C70-104EDBB9E6D2
> USER=test
> SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.y9qZ78S9dZ/Listeners
> PATH=/usr/local/Qt/5.14.1/clang_64/bin:/usr/local/opt/openssl@1.1/bin:/opt/local/bin:/opt/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
> PWD=/users/test/documents/github/retroshare
> LANG=de_DE.UTF-8
> XPC_FLAGS=0x0
> XPC_SERVICE_NAME=0
> SHLVL=1
> HOME=/Users/test
> LOGNAME=test
> _=/usr/bin/printenv

On compiling I get the the error that the -lssl not found.
```
itHub/RetroShare/libbitdht/src/lib -lbitdht -ludp-discovery -lsqlite3 -lpthread -lssl -lcrypto -lz -lminiupnpc -framework CoreFoundation -framework Security -framework Carbon -L/Applications/Xcode.app/Contents/Developer/usr/bin -framework QtMacExtras -framework QtPrintSupport -framework QtWidgets -framework QtMultimedia -framework QtGui -framework AppKit -framework Metal -framework QtNetwork -framework QtXml -framework QtCore -framework DiskArbitration -framework IOKit -framework OpenGL -framework AGL   
ld: library not found for -lssl
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

maybe i forget something?  any help is welcome

**Comment by SMillerDev** [comment]
`-lssl` isn't in the list of suggestions. It's trying to pick up openssl from the system (Which it can't because macOS doesn't have openssl)
